rs/nns/integration_tests/src/create_service_nervous_system.rs

```diff
@@ -181,12 +181,115 @@ fn test_several_proposals() {
     assert_eq!(snses.len(), 1, "{:#?}", snses);
 }
 
+#[test]
+fn test_nf_is_not_permitted() {
+    // Step 1: Prepare the world.
+
+    let state_machine = state_machine_builder_for_nns_tests().build();
+
+    // Step 1.1: Boot up NNS.
+    let nns_init_payload = NnsInitPayloadsBuilder::new()
+        .with_initial_invariant_compliant_mutations()
+        .with_test_neurons_fund_neurons(100_000_000_000_000)
+        .with_sns_dedicated_subnets(state_machine.get_subnet_ids())
+        .with_sns_wasm_access_controls(true)
+        // TODO: Delete this once the SNS_WASM canister takes any requests
+        // coming from NNS governance.
+        .with_sns_wasm_allowed_principals(vec![PrincipalId::from(GOVERNANCE_CANISTER_ID)])
+        .build();
+    // Note that this uses governance with cfg(features = "test") enabled.
```

Comment:
Is this right? I thought the point here was to use release build of Governance WASM.

Reply:
No, the comments weren't correct, and some were copied / pasted.  Cleaning them up.